pkgs/tools/misc/popsicle/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, rustc
+, cargo
+, rustPlatform
+, pkgconfig
+, dbus
+, glib
+, cairo
+, pango
+, atk
+, gdk-pixbuf
+, gtk3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "popsicle";
+  version = "b02ebf5f2e6c18777453ca9a144d69689a6fa901";
+  src = fetchFromGitHub {
```

Comment:
We have to use a relatively new commit, because all recent releases of popsicle don't actually build... Not just don't build in nix, they just don't compile anywhere

I did test all features of popsicle/popsicle-gtk on this commit and they work correctly.

The version and Sha should be fixed in the newest commit.